CONTRIBUTING.md

```diff
@@ -1,6 +1,9 @@
+![Lostrómos logo](docs/images/logo.png)
 # Contribution Guidelines
 
-## Security issues
+## <a name="bugs"></a>Report a Bug
```

Comment:
What does this end up doing? My html/markdown is weak in this area, so I'm not sure how that is different from

## Report a Bug

Reply:
It's so I can reference the tag from another page.